charts/passbolt-operator/templates/deployment.yaml

```diff
@@ -28,7 +28,32 @@ spec:
       serviceAccountName: {{ include "passbolt-operator.serviceAccountName" . }}
       terminationGracePeriodSeconds: 10
       containers:
-        - name: {{ .Chart.Name }}
+        {{- if .Values.monitoring.enabled }}
+        - name: kube-rbac-proxy
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - "ALL"
+          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.0
+          args:
+          - "--secure-listen-address=0.0.0.0:8443"
+          - "--upstream=http://127.0.0.1:8080/"
+          - "--logtostderr=true"
+          - "--v=0"
+          ports:
+          - containerPort: 8443
+            protocol: TCP
+            name: metrics
+          resources:
+            limits:
+              cpu: 500m
+              memory: 128Mi
+            requests:
+              cpu: 5m
+              memory: 64Mi
```

Comment:
Keep memory request and limit at the same value to prevent running OOM on the node (we maybe waste some memory here but this gives us overall greater stability if we don't overcommit on RAM).

Reply:
I have changed it a bit so that the resources can be customized by the user.